course-outline.md

```diff
@@ -0,0 +1,105 @@
+
+# Suggested Learning Objectives / Outline of Each Section
+
+The overall idea of this course can be articulated as introducing students to enough tooling knowledge that they are comfortable creating and maintaining an online coding/technical resource, such as a JupyterBook. This will cover the skills they need to work with any type of literate coding document under version control:
+
+- Use the shell to move around on their computer
+- Use git and github to version control their project
+- Have a good project organization
+- Install jupytet-book in an isolated virutal env
+- Create reproducible notebook documents and version control them
+- Build and upload a Jupyter Book
+
+## 1. Intro
+
+1. Introduction to the toolbox, give an overview of what each tool does and where it fits in the toolchain
+2. Intro to VS Code
```

Comment:
Maybe contrast to some jlab basics as they learned in Python programming course?

Reply:
Make sure to define the purpose of teaching VScode. text editor, merge tool, edit script?

Reply:
VS code does merge conflicts better than jlab git addon as discussed at the meeting today so that will be one of the main purposes together with text editor. Python scripts still TBD